mint-choc-chip-skyblade/sshd-rando
# Tracker: Display computed tooltips for entrances

## What does this address?

When tooltips were introduced by Robojumper, functionality was added to calculate tooltips for entrances, but they were never displayed on the tracker. 

## How did/do you test these changes?

- Load tracker with any entrance randomization settings (in my case, random starting location and random starting statues).
- Verify that the entrances in the Everything Discovered section have correct tooltips (Nothing for starting location, the respective tablet for each pillar)

## Notes

- Moves the helper functions that were in `tracker_location_label.py` to be part of `tooltips.py` so they could be used by both the location and entrance label components with an import.
